androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/utils/TrackQualityItem.kt

```diff
@@ -0,0 +1,38 @@
+package com.profusion.androidenhancedvideoplayer.utils
+
+import androidx.compose.runtime.Stable
+import androidx.media3.common.Format
+import androidx.media3.common.TrackSelectionParameters
+import androidx.media3.common.util.UnstableApi
+
+@UnstableApi
+@Stable
+class TrackQualityItem(
+    val format: Format? = null,
+    val autoLabel: String? = null,
+    val builder: TrackSelectionParameters.Builder
```

Comment:
```suggestion
    val builder: TrackSelectionParameters
```

Reply:
Yeah. It is better to pass the builded object instead of a builder.